app/controllers/admin/registries_controller.rb

```diff
@@ -7,11 +7,31 @@ class Admin::RegistriesController < Admin::BaseController
   # GET /admin/registries/
   def index
     @registries = Registry.all
+
+    redirect_to new_admin_registry_url unless Registry.any?
   end
 
   # GET /admin/registries/new
   def new
     @registry = Registry.new
+    @can_change_hostname = true
+  end
+
+  # GET /admin/registries/reachable
+  def reachable
```

Comment:
Are you really using this ? (you are using the API on the client side, right?)

Reply:
I was but now we have the validate endpoint.

Reply:
Yeah, please use the validate endpoint instead 👍